Rooster Teeth/dist/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "author": {
+      "name": "qwerty142",
+      "id": "455167347464863746"
+    },
+    "service": "Rooster Teeth",
+    "description": {
+      "en": "Rooster Teeth is a pioneering media and entertainment company responsible for some of the biggest online series in history, such as the award-winning and longest-running web series, Red vs. Blue"
+    },
+    "url": "roosterteeth.com",
+    "version": "1.0.1",
+    "logo": "https://i.imgur.com/sxZnjmY.png",
+    "thumbnail": "https://i.imgur.com/BwJU788.png",
+    "color": "#2c2420",
+    "tags": ["video", "media","streaming"],
```

Comment:
fixes the spacing between media and streaming tags

Reply:
fixed